src/content/docs/configuration/sched-ext.md

```diff
@@ -9,16 +9,18 @@ build another kernel for a different scheduler.
 
 - Planned release for being an official kernel feature: 6.12
 
-##  Installing a Kernel with sched-ext support
+## Installing a Kernel with sched-ext support
 
 CachyOS provides kernels, which have OOB support for the sched-ext framework.
 Following kernels are supported:
+
 - linux-cachyos (default kernel)
 - linux-cachyos-sched-ext (latest Stable release)
 - linux-cachyos-sched-ext-debug (This is mainly for developers to develop and work on sched-ext)
 - linux-cachyos-rc (latest testing release with the latest features)
 
-You can simply check with following command, if your kernel supports sched-ext:
+You can simply check if your kernel supports sched-ext with the following command:
+
 ```bash
 ❯ zcat /proc/config.gz | grep SCHED_CLASS_EXT
 CONFIG_SCHED_CLASS_EXT=y
```

Comment:
```suggestion
cat /sys/kernel/sched_ext/state
```

Do you think its better to check for this sysfs file instead of quering the kernel config? Having `CONFIG_SCHED_CLASS_EXT` enabled doesn't necessarily mean sched-ext support is enabled because it also depends on other kernel configs, e.g. `DEBUG_BTF` and some pahole configs. cc @ptr1337

Reply:
Hm, but using that command is for checking if sched-ext is currently being used or not instead of: Does my kernel have support for it?.

Reply:
It also works for the latter, since if the kernel doesn't have sched-ext support, the sysfs directory would **not** be there.